src/test/scala/io/iohk/ethereum/network/p2p/messages/MessagesSerializationSpec.scala

```diff
@@ -91,12 +91,14 @@ class MessagesSerializationSpec extends AnyWordSpec with ScalaCheckPropertyCheck
     }
   }
 
-  "ETH61" when {
-    val version = ProtocolVersions.ETH61
+  "support ETH61 messages as ETH63" when {
```

Comment:
Nitpick: I think the following would read better:

```
"ETH63" when {
  "encoding and decoding ETH62.NewBlockHashes" should {
    ...
  }
}
```

You could merge the above with `"support ETH62 messages as ETH63"`